### ISSUE TITLE
imaplib2 issue in docker container

### ISSUE THREAD

**Issue by rennefJ** [OfflineIMAP/offlineimap3#90]
#### General informations

- system/distribution (with version): Python 3-alpine docker image
- offlineimap version (`offlineimap -V`): 7.3.3
- Python version: 3.10.0

#### Logs, error

  File "/usr/src/offlineimap3/offlineimap/init.py", line 29, in <module>

    import offlineimap.virtual_imaplib2 as imaplib

  File "/usr/src/offlineimap3/offlineimap/virtual_imaplib2.py", line 36, in <module>

    if (int(imaplib.__release__) < _SUPPORTED_RELEASE or

AttributeError: module 'imaplib2' has no attribute '__release__'

#### Steps to reproduce the error

I am trying to get offlineimap3 running in a docker container. But when trying to run the container I always get the error message :
" if (int(imaplib.__release__) < _SUPPORTED_RELEASE or AttributeError: module 'imaplib2' has no attribute '__release__'"
I am using imaplib2 version > 3.5  as listed in the prerequisites. 
Is this a known bug or am I doing something wrong?

**Comment by rennefJ** [comment]
Hi @thekix 
I am using the latest release ﻿﻿v7.3.3 (on 11 Apr 2020 7531ac4 ) that I could find on the release page.
I tried in docker and under macOS 11.6.
I tried also with the python2 version v7.3.4. And I always get the same error message.
I am using imaplab2 v3.6

When I download the v7.3.3 tar.gz and look at init.py
in line 29 I find:
import offlineimap.virtual_imaplib2 as imaplib

What am I doing wrong? Which version of offlineimap3 should I use?

**Comment by rennefJ** [comment]
Hi @thekix 
I saw there is now release 8.0.0 but it has the same path as the version 7.3.3 I tried and did not work:
https://github.com/OfflineIMAP/offlineimap3/archive/refs/tags/
It has offlineimap3 in the path, how can I tell if it is for python3 or not?

I tried downloading the source from master as zip file.
This time I am getting a different error:
> ImportError: cannot import name 'Mapping' from 'collections' (/usr/local/lib/python3.10/collections/__init__.py)
Am I missing a dependency?
I installed dependencies that are listed (imaplib2 distro rfc6555 urllib3 certifi)

regards
joerg
